scripts/build_flex_driver

```diff
@@ -5,4 +5,4 @@ set -e
 scripts=$(dirname $0)
 
 echo "Building flex driver"
-go build -o $scripts/../bin/ubiquity $scripts/../cli.go
+go build -ldflags -s -o $scripts/../bin/ubiquity $scripts/../cmd/flex/main/cli.go
```

Comment:
This is to avoid some errors in darwin systems where the linker will invoke dsymutil to put the debug info directly into the executable for go code. This invocation of dsymutil is disabled by the linker's -ldflags -s option.